.github/workflows/daily-builds.yml

```diff
@@ -53,8 +59,20 @@ jobs:
           cmake -B ${{github.workspace}}/build -DCMAKE_BUILD_TYPE=Debug -GNinja -DCMAKE_CXX_COMPILER_LAUNCHER=ccache ${{ matrix.flags }}
           cd ${{github.workspace}}/build && pwd
           du -hcs _deps/
-      - name: Build
+      - name: Build & Run
         run: |
           cd ${{github.workspace}}/build
           ninja src/all
           ccache --show-stats
+          ./dragonfly --alsologtostderr --default_lua_flags=allow-undeclared-keys &
+      - name: Clone and build BullMQ
```

Comment:
please change BullMQ tests to run on release build

Reply:
In that case, I am adding a new job "bullmq" instead of testing bullmq in the "build" job

Reply:
decided to run the tests in same job